dotCMS/src/main/java/com/dotcms/publisher/ajax/RemotePublishAjaxAction.java

```diff
@@ -738,11 +737,36 @@ public void uploadBundle ( HttpServletRequest request, HttpServletResponse respo
             out.print( "<html><head><script>isLoaded = true;</script></head><body><textarea>{'status':'success'}</textarea></body></html>" );
 
         } catch ( DotPublisherException e ) {
+            Logger.error(this, String.format("An error occurred when uploading bundle '%s'", bundleName), e);
             out.print( "<html><head><script>isLoaded = true;</script></head><body><textarea>{'status':'error'}</textarea></body></html>" );
         }
 
     }
 
+    /**
+     * Checks the name of the bundle file coming from the request to make sure that it is correct. For example, in
+     * Microsoft Edge on Windows 10, the name includes the absolute path to the file instead of the file name only. We
+     * need to strip the absolute path away.
+     *
+     * @param bundleName The name of the bundle file.
+     *
+     * @return The sanitized bundle file name.
+     */
+    private String sanitizeBundleName(String bundleName) throws DotPublisherException {
```

Comment:
@jcastro-dotcms Place somewhere else? Maybe a util? BundleUtil? and a unit test should be nice

Reply:
Done, moved to the `BundleUtil`

Reply:
Also de unit test was created.